### PR TITLE
Add coach 1 to master

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)

See the changelog of release v1.0.0 for additional infos.

**Other information**:
- The version has been updated to 1.0.0

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks good enough for v.1.0.0 - 🔥👀

**Comment by kleberbaum** [comment]
> **Checklist:**
> 
> * [x]  Have you added an explanation of what your changes do and why you'd like them to be included?
> * [ ]  Have you updated or added documentation for the change?
> * [x]  Have you tested your changes with successful results?
> 
> **Type of Changes:**
> 
> * [ ]  Bug fix (non-breaking change which fixes an issue)
> * [x]  New feature (non-breaking change which adds functionality)
> * [x]  Breaking change (fix or feature that would cause existing functionality to change)
> 
> **What is the current behavior? (link to any open issues here)**
> 
> * The current web app has no functionality.
> 
> **What is the new behavior (if this is a feature change)?**
> 
> * Cats can now be created.
> 
> **Other information**:
> 
> * The version has been updated to 1.0.0

There should be at least a few linked issues.

**Review by sweggerat — APPROVED** [review]
Copyright dates may be updated

File: src/components/molecules/charts/ResultChart/index.jsx [hunk 1]
```diff
@@ -9,46 +9,49 @@ import { Radar } from "react-chartjs-2";
...
+          },
+          ticks: {
+            beginAtZero: true,
+            display: false,
+          }
+        }]
+      }
     },
     dataRadarPlugins: [{
         beforeInit: function (chart) {
```
Comment (kleberbaum) [inline src/components/molecules/charts/ResultChart/index.jsx:62]: Please fix false indentation. (2 spaces)

File: src/components/molecules/charts/ResultChart/index.jsx [hunk 2]
```diff
@@ -61,43 +64,33 @@ class ResultChart extends React.Component{
...
-      console.log("Got data",this.props.data);
+    if(this.props.data){
       let labels, data;
       labels = [];
       data = [];
 
-      this.props.data.data.map((item) => {
-        console.log(item);
+      // Get value and label
+      Object.keys(this.props.data).map((keyName, i) => {
```
Comment (kleberbaum) [inline src/components/molecules/charts/ResultChart/index.jsx:80]: Please fix false indentation. (2 spaces)

File: src/components/organisms/FormCat/index.jsx [hunk 3]
```diff
@@ -0,0 +1,744 @@
...
+          ? this.state.web_picture_quality
+          : 50,
+        web_wording: this.state.web_wording ? this.state.web_wording : 50,
+      };
+      // Get web average
+      const web_avg =
+        this.getSum(web) / (analysis.web.length - 1) -
+        (this.state.web_performance > 2000
+          ? this.state.web_performance / 200
+          : 0);
```
Comment (kleberbaum) [inline src/components/organisms/FormCat/index.jsx:108]: Please add a newline.

File: src/components/organisms/FormCat/index.jsx [hunk 4]
```diff
@@ -0,0 +1,744 @@
...
+        facebook_contact: this.state.facebook_contact
+          ? this.state.facebook_contact
+          : 50,
+        facebook_likes: this.state.facebook_likes
+          ? this.state.facebook_likes
+          : 50,
+        facebook_regularity: this.state.facebook_regularity
+          ? this.state.facebook_regularity
+          : 50,
+      };
```
Comment (kleberbaum) [inline src/components/organisms/FormCat/index.jsx:123]: Please add a newline.

File: src/components/organisms/FormCat/index.jsx [hunk 5]
```diff
@@ -0,0 +1,744 @@
...
+      // Get Facebook average
+      const facebook_avg = this.getSum(facebook) / analysis.facebook.length;
```
Comment (kleberbaum) [inline src/components/organisms/FormCat/index.jsx:125]: Please add a newline.

File: src/components/organisms/FormCat/index.jsx [hunk 6]
```diff
@@ -0,0 +1,744 @@
...
+        instagram_followers: this.state.instagram_followers
+          ? this.state.instagram_followers
+          : 50,
+        instagram_picture_quality: this.state.instagram_picture_quality
+          ? this.state.instagram_picture_quality
+          : 50,
+        instagram_regularity: this.state.instagram_regularity
+          ? this.state.instagram_regularity
+          : 50,
+      };
```
Comment (kleberbaum) [inline src/components/organisms/FormCat/index.jsx:140]: Please add a newline.

File: src/components/organisms/FormCat/index.jsx [hunk 7]
```diff
@@ -0,0 +1,744 @@
...
+      const instagram_avg = this.getSum(instagram) / analysis.instagram.length;
+      // Corporate Identity
+      const corporate = {
+        corporate_consistency: this.state.corporate_consistency
+          ? this.state.corporate_consistency
+          : 50,
+        corporate_logo: this.state.corporate_logo
+          ? this.state.corporate_logo
+          : 50,
+      };
```
Comment (kleberbaum) [inline src/components/organisms/FormCat/index.jsx:151]: Please add a newline.

File: src/components/organisms/FormCat/index.jsx [hunk 8]
```diff
@@ -0,0 +1,744 @@
...
+      // Get Corporate Identity average
+      const corporate_avg = this.getSum(corporate) / analysis.corporate.length;
+      // Image
+      const image = {
+        image_portfolio: this.state.image_portfolio
+          ? this.state.image_portfolio
+          : 50,
+        image_film: this.state.image_film ? this.state.image_film : 50,
+      };
```
Comment (kleberbaum) [inline src/components/organisms/FormCat/index.jsx:160]: Please add a newline.

File: src/components/organisms/FormCat/index.jsx [hunk 9]
```diff
@@ -0,0 +1,744 @@
...
+          : 50,
+        web_design: this.state.web_design ? this.state.web_design : 50,
+        web_analysis: this.state.web_analysis ? this.state.web_analysis : 50,
+        web_contact: this.state.web_contact ? this.state.web_contact : 50,
+        web_legal: this.state.web_legal ? this.state.web_legal : 50,
+        web_picture_quality: this.state.web_picture_quality
+          ? this.state.web_picture_quality
+          : 50,
+        web_wording: this.state.web_wording ? this.state.web_wording : 50,
+      };
```
Comment (kleberbaum) [inline src/components/organisms/FormCat/index.jsx:102]: Please add a newline.

File: src/components/organisms/FormCat/index.jsx [hunk 10]
```diff
@@ -0,0 +1,744 @@
...
+          : 50,
+        instagram_picture_quality: this.state.instagram_picture_quality
+          ? this.state.instagram_picture_quality
+          : 50,
+        instagram_regularity: this.state.instagram_regularity
+          ? this.state.instagram_regularity
+          : 50,
+      };
+      // Get Instagram average
+      const instagram_avg = this.getSum(instagram) / analysis.instagram.length;
```
Comment (kleberbaum) [inline src/components/organisms/FormCat/index.jsx:142]: Please add a newline.

File: src/components/organisms/FormCat/index.jsx [hunk 11]
```diff
@@ -0,0 +1,744 @@
...
+      const corporate_avg = this.getSum(corporate) / analysis.corporate.length;
+      // Image
+      const image = {
+        image_portfolio: this.state.image_portfolio
+          ? this.state.image_portfolio
+          : 50,
+        image_film: this.state.image_film ? this.state.image_film : 50,
+      };
+      // Get Image average
+      const image_avg = this.getSum(image) / analysis.image.length;
```
Comment (kleberbaum) [inline src/components/organisms/FormCat/index.jsx:162]: Please add a newline.

File: src/components/organisms/FormCat/index.jsx [hunk 12]
```diff
@@ -0,0 +1,744 @@
...
+      // SEO
+      const seo = {
+        seo_code: this.state.seo_code ? this.state.seo_code : 50,
+        seo_domain: this.state.seo_domain ? this.state.seo_domain : 50,
+        seo_https: this.state.seo_https ? this.state.seo_https : 50,
+        seo_meta: this.state.seo_meta ? this.state.seo_meta : 50,
+        seo_traceability: this.state.seo_traceability
+          ? this.state.seo_traceability
+          : 50,
+      };
```
Comment (kleberbaum) [inline src/components/organisms/FormCat/index.jsx:172]: Please add a newline.

File: src/components/organisms/FormCat/index.jsx [hunk 13]
```diff
@@ -0,0 +1,744 @@
...
+
+      console.log(userobject);
+
+      this.props.createCat(userobject);
+    }
+  };
+
+  render() {
+    const { auth, profile, catDetails } = this.props;
```
Comment (kleberbaum) [inline src/components/organisms/FormCat/index.jsx:229]: Please add a newline.

File: src/index.scss [hunk 14]
```diff
@@ -1,10 +1,10 @@
...
+@import "utilities/variables";
 // Global CSS
-@import 'utilities/base';
+@import "utilities/base";
 // Fonts
-@import 'utilities/fonts';
+@import "utilities/fonts";
 
 /** 
  * SPDX-License-Identifier: (EUPL-1.2)
```
Comment (kleberbaum) [inline src/index.scss:10]: Update to (c) 2020.

File: src/store/actions/postActions.js [hunk 15]
```diff
@@ -156,29 +156,29 @@ export const loadAllPosts = (amount) => {
...
+            dispatch({ type: "LOAD_ERROR", err });
         });
     }
 }
 
 /** 
  * SPDX-License-Identifier: (EUPL-1.2)
- * Copyright © 2019 Christian Aichner
+ * Copyright © 2020 Christian Aichner
  */
```
Comment (kleberbaum) [inline src/store/actions/postActions.js]: Please add a newline.

File: src/store/actions/userActions.js [hunk 16]
```diff
@@ -4,34 +4,34 @@ export const getUser = (uid) => {
...
-        dispatch({ type: 'USERLOAD_CLEAR' });
+        dispatch({ type: "USERLOAD_CLEAR" });
     }
 }
 
 /** 
  * SPDX-License-Identifier: (EUPL-1.2)
- * Copyright © 2019 Christian Aichner
+ * Copyright © 2020 Christian Aichner
  */
```
Comment (kleberbaum) [inline src/store/actions/userActions.js]: Please add a newline.

File: src/utilities/_base.scss [hunk 17]
```diff
@@ -135,6 +135,9 @@ body{
 
 .btn{
```
Comment (kleberbaum) [inline src/utilities/_base.scss]: Please fix false indentation. (2 spaces)

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: src/components/organisms/FormCat/index.jsx (hunk 3 above)
Comment (Aichnerc) [reply src/components/organisms/FormCat/index.jsx:108]: The VSCode Prettier does not like a new line.

File: src/components/organisms/FormCat/index.jsx (hunk 4 above)
Comment (Aichnerc) [reply src/components/organisms/FormCat/index.jsx:123]: The VSCode Prettier does not like a new line.

File: src/components/organisms/FormCat/index.jsx (hunk 13 above)
Comment (Aichnerc) [reply src/components/organisms/FormCat/index.jsx:229]: The VSCode Prettier does not like a new line.

File: src/components/organisms/FormCat/index.jsx (hunk 6 above)
Comment (Aichnerc) [reply src/components/organisms/FormCat/index.jsx:140]: The VSCode Prettier does not like a new line.

File: src/components/organisms/FormCat/index.jsx (hunk 5 above)
Comment (Aichnerc) [reply src/components/organisms/FormCat/index.jsx:125]: The VSCode Prettier does not like a new line.

File: src/components/organisms/FormCat/index.jsx (hunk 7 above)
Comment (Aichnerc) [reply src/components/organisms/FormCat/index.jsx:151]: The VSCode Prettier does not like a new line.

File: src/components/organisms/FormCat/index.jsx (hunk 8 above)
Comment (Aichnerc) [reply src/components/organisms/FormCat/index.jsx:160]: The VSCode Prettier does not like a new line.

File: src/components/organisms/FormCat/index.jsx (hunk 9 above)
Comment (Aichnerc) [reply src/components/organisms/FormCat/index.jsx:102]: The VSCode Prettier does not like a new line.

File: src/components/organisms/FormCat/index.jsx (hunk 10 above)
Comment (Aichnerc) [reply src/components/organisms/FormCat/index.jsx:142]: The VSCode Prettier does not like a new line.

File: src/components/organisms/FormCat/index.jsx (hunk 11 above)
Comment (Aichnerc) [reply src/components/organisms/FormCat/index.jsx:162]: The VSCode Prettier does not like a new line.

File: src/components/organisms/FormCat/index.jsx (hunk 12 above)
Comment (Aichnerc) [reply src/components/organisms/FormCat/index.jsx:172]: The VSCode Prettier does not like a new line.

**Pull request merged**